quinn-proto/src/connection.rs

```diff
@@ -2494,12 +2425,7 @@ where
             }
             sent.max_stream_data.insert(id);
             let max = rs.bytes_read + self.config.stream_receive_window;
-            trace!(
-                self.log,
-                "MAX_STREAM_DATA: {stream} = {max}",
-                stream = id,
-                max = max
-            );
+            trace!(stream = %id, max = max, "MAX_STREAM_DATA",);
```

Comment:
Is the trailing comma mandatory here? Should we file a bug about that?

Reply:
Nah, just a refactoring artifact. Fixed.